cardinal/server/handler/debug.go

```diff
@@ -1,23 +1,13 @@
 package handler
 
 import (
-	"encoding/json"
-
 	"github.com/gofiber/fiber/v2"
 
-	"pkg.world.dev/world-engine/cardinal/search/filter"
 	servertypes "pkg.world.dev/world-engine/cardinal/server/types"
 	"pkg.world.dev/world-engine/cardinal/types"
 )
 
-type DebugStateRequest struct{}
-
-type debugStateElement struct {
-	ID         types.EntityID             `json:"id"`
-	Components map[string]json.RawMessage `json:"components" swaggertype:"object"`
-}
-
-type DebugStateResponse []debugStateElement
+type DebugStateResponse = types.DebugStateResponse
```

Comment:
This is done to satisfy swagger.

Reply:
@smsunarto I messed with this a bit. --parseDependency and --parseInternal don't work. Using --dir didn't work for me either. 

I think type aliases work fine here even without swagger. We did follow this pattern in cardinal where we would alias other types that were coming from other packages and create a new alias in cardinal. I would recommend leaving this as is unless you have some other ideas.